src/events/event.controller.ts

```diff
@@ -16,6 +16,11 @@ export class EventController {
   ) {}
 
   @Get('/dapr/subscribe')
+  /**
```

Comment:
docs should be placed above the decorator

Reply:
Ok. I couldnt find any reference regarding this but understand your approach.